pom.xml

```diff
@@ -147,12 +147,23 @@
                 <version>1.5.0-SNAPSHOT</version>
             </dependency>
             <dependency>
+                <groupId>com.netflix.hystrix</groupId>
+                <artifactId>hystrix-core</artifactId>
+                <version>1.3.16</version>
+            </dependency>
+            <dependency>
                 <groupId>com.redhat.lightblue</groupId>
                 <artifactId>lightblue-core-test</artifactId>
                 <version>1.5.0-SNAPSHOT</version>
                 <scope>test</scope>
             </dependency>
             <dependency>
+                <groupId>com.redhat.lightblue.mongo</groupId>
+                <artifactId>lightblue-mongo-test</artifactId>
```

Comment:
This ties the version of the referenced project to the version of the root pom. Those two things should not be tied together. Allows for creating a dependency in a sub-project to a version of another entity whose version number is determined by the root pom version number. Please remove this.